<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaderboardUpdateWindow</class>
 <widget class="QDialog" name="LeaderboardUpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Osiagnales jeden z najlepszych wynikow:</string>
   </property>
  </widget>
  <widget class="QLabel" name="namePromptLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gratulacje! Podaj swoje imie:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>140</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
